package.json

```diff
@@ -5,7 +5,8 @@
   "main": "app.js",
   "scripts": {
     "start": "node app.js",
-    "lint": "eslint --ext .js ."
+    "lint": "npx @biomejs/biome check *.js",
+    "lint:fix": "npx @biomejs/biome check --write *.js"
```

Comment:
This change is a similar change that's found in https://github.com/slack-samples/bolt-ts-starter-template/pull/131 - open to making changes here!